src/gt4py/next/otf/compilation/build_systems/compiledb.py

```diff
@@ -321,14 +346,20 @@ def _cc_create_compiledb(
     assert compile_db
 
     for entry in compile_db:
-        entry["directory"] = entry["directory"].replace(str(path), "$SRC_PATH")
+        relative_path_from_build_dir = _relative_path_to_parent(
+            pathlib.Path(entry["directory"]), path
+        )
+
+        # directory relative to current root directory
+        entry["directory"] = entry["directory"].replace(str(path), ".")
```

Comment:
replace by a check that the directory is only one-level deep (less general, but we currently don't have the general case).